findme-location-riddles/src/entities/Guess.py

```diff
@@ -0,0 +1,18 @@
+from pydantic import BaseModel, field_validator, ValidationError
+from typing import List
+from decimal import Decimal
+
+
+class Guess(BaseModel):
+    user_id: str
+    guess: List[Decimal]
+
+    @field_validator('guess')
+    def validate_coordinates(cls, v):
+        if len(v) != 2:
+            raise ValidationError('Coordinate should contain 2 values: latitude and longitude')
```

Comment:
In the frontend we are using openLayers for the map. By default open layers is using the EPSG:3867 format

>This projection is a good choice if you want a map which shows the whole world, and you may need to have this projection if you want to e.g. use the OpenStreetMap or Bing tiles.
 https://openlayers.org/doc/faq.html

 which has other validation rules. We could convert that and use typical coordinate system but I am not sure if it might be easier to always deal with the native format of the map in the frontend